examples/interactions.js

```diff
@@ -1,19 +1,19 @@
-const { Client, InteractionTypes, MessageFlags, ComponentTypes, ApplicationCommandTypes } = require("oceanic.js");
+const { Client, InteractionTypes, ComponentTypes, ApplicationCommandTypes } = require("oceanic.js");
```

Comment:
```suggestion
const { Client, InteractionTypes, MessageFlags, ComponentTypes, ApplicationCommandTypes } = require("oceanic.js");
```

This import may be "unused", but it's here on purpose
It's in a commented example